### PR TITLE
[RA2 Ch2] req.inf.vir.01 & req.inf.phy.01

### LINKED ISSUE #606: [RA2 Core] Requirements: The Architecture **must** support the capability for Containers to consume virtualised compute, storage and network resources.
Ambiguous. It suggests K8s workloads would be able to somehow request/manage virtualised resources or use them raw (e.g. by accessing the VM/BM hardware bypassing the OS), which would be a layering and abstraction violation. What's probably meant here is to consume the compute (threads, memory, ...), storage (block/char devices), and network (sockets, interface devices) abstractions provided by the host OS?

_Originally posted by @fzdarsky in https://github.com/cntt-n/CNTT/pull/452/files_

Yes - the intent is to ensure that those host OSs can be virtualised or physical/BM without falling foul of something we've written into the RA2. Would the following work better?

| req.inf.vir.01 | Virtualisation | The Architecture must support the capability for Containers to consume the abstracted compute, storage and network resources provided by Host Operating Systems that are running within a virtual machine |

_Originally posted by @tomkivlin in https://github.com/cntt-n/CNTT/pull/452/files_

### LINKED ISSUE #607: [RA2 Core] Requirements: The Architecture **must** support the capability for Containers to consume physical (aka bare metal) compute, storage and network resources.
Same as above (#606)

_Originally posted by @fzdarsky in https://github.com/cntt-n/CNTT/pull/452/files_

As above, would the following work?

| req.inf.phy.01 | Physical Nodes | The Architecture must support the capability for Containers to consume the abstracted compute, storage and network resources provided by Host Operating Systems that are running within a physical/bare metal machine. |

_Originally posted by @tomkivlin in https://github.com/cntt-n/CNTT/pull/452/files_

### PR DESCRIPTION
**Pull request by tomkivlin**
Added new wording for these requirements.

Fixes #607 
Fixes #606 

@fzdarsky does this wording work for you?  I can't add you as a reviewer for some reason.

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter02.md [hunk 1]
```diff
@@ -82,14 +82,15 @@ Reference to "Architecture" in this chapter refers to the NFVI Hardware (e.g. ph
 | `req.inf.ntw.16` | Network | The Architecture **should**  support co-exisence of multiple Conatiner Network Interface (CNI). |
 | `req.inf.acc.01` | Acceleration | The Architecture **should** support Application Specific Acceleration. |
 | `req.inf.acc.02` | Acceleration | The Architecture **should** support NFVI Acceleration (such as SmartNICs). |
```
Comment (TamasZsiros) [inline doc/ref_arch/kubernetes/chapters/chapter02.md]: I am not sure what NFVI implies here. The need to have underlying RA1 in order to use SmartNICs? If that is the case I do not agree, I think one of the benefits of SmartNICs can be the omission of any sort of IAAS while still maintaining network automation. 

If that is not implied then I would just say "The Architecture **should** support acceleration and/or network automation via SmartNICs".

**Comment by tomkivlin** [comment]
> I propose to discuss changing the SmartNIC requirement before approval.
> "The Architecture **should** support acceleration and/or network automation via SmartNICs"

That wasn't added by this PR - can we keep reviews about the changes made please? I think a separate PR to remove would be preferable, if that's what we want.

**Comment by TamasZsiros** [comment]
Sorry, my bad, should have created a separate PR for my issue, I realized that after leaving the comment.

**Comment by rabiabdel** [comment]
This PR will be labeled as "**idle**" and it will be marked as "**force merged**" during TSC call if no more reviews/approvals received by then. It will be automatically merged in 5 days.

**Pull request merged**